utils/sql-db-seeder/src/fee-record-payment-group/fee-record-payment-group.seeder.ts

```diff
@@ -75,6 +75,22 @@ export class FeeRecordPaymentGroupSeeder {
     return this;
   }
 
+  private async saveFacilityUtilisationData(dataSource: DataSource): Promise<void> {
```

Comment:
It looks to me like this method is already on `main` - has something gone wrong with merging `main` in at some point?